core/dbt/task/clean.py

```diff
@@ -39,6 +40,14 @@ def run(self):
 
         fire_event(FinishedCleanPaths())
 
+    @classmethod
+    def from_args(cls, args) -> "CleanTask":
```

Comment:
Restoring this method for the purposes of backwards-compatability for testing. Removed in [`Remove from_args from CleanTask`](https://github.com/dbt-labs/dbt-core/pull/6369/commits/9e36c32cc504dba21790a4ede6ad45e129664fd8)